From: Stefano Rivera <stefanor@debian.org>
Date: Sun, 27 Dec 2020 12:05:10 -0800
Subject: Use youtube-nocookie.com for embedded videos

youtube-nocookie is a little more privacy friendly than regular
youtube and is a minor change.

--- dask.orig/docs/source/array.rst
+++ dask/docs/source/array.rst
@@ -29,7 +29,7 @@
 
    <iframe width="560"
            height="315"
-           src="https://www.youtube.com/embed/9h_61hXCDuI"
+           src="https://www.youtube-nocookie.com/embed/9h_61hXCDuI"
            style="margin: 0 auto 20px auto; display: block;"
            frameborder="0"
            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
--- dask.orig/docs/source/bag.rst
+++ dask/docs/source/bag.rst
@@ -31,7 +31,7 @@
 
    <iframe width="560"
            height="315"
-           src="https://www.youtube.com/embed/-qIiJ1XtSv0"
+           src="https://www.youtube-nocookie.com/embed/-qIiJ1XtSv0"
            style="margin: 0 auto 20px auto; display: block;"
            frameborder="0"
            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
--- dask.orig/docs/source/dataframe.rst
+++ dask/docs/source/dataframe.rst
@@ -193,7 +193,7 @@
 
   <iframe width="560"
           height="315"
-          src="https://www.youtube.com/embed/AT2XtFehFSQ"
+          src="https://www.youtube-nocookie.com/embed/AT2XtFehFSQ"
           style="margin: 0 auto 20px auto; display: block;"
           frameborder="0"
           allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
--- dask.orig/docs/source/delayed.rst
+++ dask/docs/source/delayed.rst
@@ -32,7 +32,7 @@
 
    <iframe width="560"
            height="315"
-           src="https://www.youtube.com/embed/SHqFmynRxVU"
+           src="https://www.youtube-nocookie.com/embed/SHqFmynRxVU"
            style="margin: 0 auto 20px auto; display: block;"
            frameborder="0"
            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
--- dask.orig/docs/source/diagnostics-distributed.rst
+++ dask/docs/source/diagnostics-distributed.rst
@@ -57,7 +57,7 @@
 
     <iframe width="560"
             height="315"
-            src="https://www.youtube.com/embed/nTMGbkS761Q"
+            src="https://www.youtube-nocookie.com/embed/nTMGbkS761Q"
             frameborder="0"
             allow="autoplay; encrypted-media"
             allowfullscreen>
--- dask.orig/docs/source/futures.rst
+++ dask/docs/source/futures.rst
@@ -13,7 +13,7 @@
 
    <iframe width="560"
            height="315"
-           src="https://www.youtube.com/embed/07EiCpdhtDE"
+           src="https://www.youtube-nocookie.com/embed/07EiCpdhtDE"
            style="margin: 0 auto 20px auto; display: block;"
            frameborder="0"
            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
@@ -468,7 +468,7 @@
 
    <iframe width="560"
            height="315"
-           src="https://www.youtube.com/embed/Q-Y3BR1u7c0"
+           src="https://www.youtube-nocookie.com/embed/Q-Y3BR1u7c0"
            style="margin: 0 auto 20px auto; display: block;"
            frameborder="0"
            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
--- dask.orig/docs/source/presentations.rst
+++ dask/docs/source/presentations.rst
@@ -5,7 +5,7 @@
 
    <iframe width="560"
            height="315"
-           src="https://www.youtube.com/embed/nnndxbr_Xq4"
+           src="https://www.youtube-nocookie.com/embed/nnndxbr_Xq4"
            style="margin: 0 auto 20px auto; display: block;"
            frameborder="0"
            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
@@ -15,11 +15,11 @@
 -------------
 
 `Dask Tutorial <https://tutorial.dask.org>`__ provides an overview of Dask and is typically delivered in 3 hours.
-See `Parallel and Distributed Computing in Python with Dask <https://www.youtube.com/watch?v=EybGGLbLipI>`__ for the
+See `Parallel and Distributed Computing in Python with Dask <https://www.youtube-nocookie.com/watch?v=EybGGLbLipI>`__ for the
 latest Dask Tutorial recording from SciPy 2020.
 
 Dask YouTube channel
 --------------------
-You can find lots of videos about Dask on the `Dask YouTube channel <https://www.youtube.com/c/dask-dev>`__
+You can find lots of videos about Dask on the `Dask YouTube channel <https://www.youtube-nocookie.com/c/dask-dev>`__
 
 .. contents:: :local:
--- dask.orig/docs/source/best-practices.rst
+++ dask/docs/source/best-practices.rst
@@ -18,7 +18,7 @@
 
 .. raw:: html
 
-    <iframe width="560" height="315" src="https://www.youtube.com/embed/Gvuf6gSGA9M" title="YouTube video player" frameborder="0" style="margin: 0 auto 20px auto; display: block;" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/Gvuf6gSGA9M" title="YouTube video player" frameborder="0" style="margin: 0 auto 20px auto; display: block;" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 
 Start Small
--- dask.orig/docs/source/dashboard.rst
+++ dask/docs/source/dashboard.rst
@@ -8,7 +8,7 @@
 
     <iframe width="560"
             height="315"
-            src="https://www.youtube.com/embed/N_GqzcuGLCY"
+            src="https://www.youtube-nocookie.com/embed/N_GqzcuGLCY"
             frameborder="0"
             allow="autoplay; encrypted-media"
             style="margin: 0 auto 20px auto; display: block;"
